webrender/src/render_task.rs

```diff
@@ -283,6 +283,7 @@ pub struct RenderTask {
     pub children: Vec<RenderTaskId>,
     pub kind: RenderTaskKind,
     pub clear_mode: ClearMode,
+    pub pass_index: usize,
```

Comment:
This could use the ```RenderPassIndex``` mentioned above. If we make it ```Option<RenderPassIndex>``` we can ensure that when the later code reads the value, it has been correctly assigned to a pass.